scipy/sparse/_compressed.py

```diff
@@ -624,8 +648,8 @@ def _maximum_minimum(self, other, npop, op_name, dense_check):
                 return mat
         elif isdense(other):
             return npop(self.todense(), other)
-        elif issparse(other):
-            return self._binopt(other, op_name)
+        elif issparse(other):       
+            return self._binopt(other, op_name) 
```

Comment:
Looks like added spaces at end of line.  That happens elsewhere too.

Reply:
Removed from there and a few other places